scripts/Muon/GUI/ElementalAnalysis/elemental_analysis.py

```diff
@@ -364,7 +367,29 @@ def select_data_file(self):
                 self.ptable.select_element(element)
             else:
                 self._remove_element_lines(element)
-        # self._generate_element_data()
+        self._update_checked_data()
+        #self._generate_element_data()
+
+    def _update_checked_data(self):
```

Comment:
Can a new test be added to check if this function is being called and calling the correct things when `subplot_removed` is called?